controllers/component_build_controller_initial_build.go

```diff
@@ -38,6 +44,15 @@ import (
 	ctrllog "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
+const (
+	pacShaLabelName          = "pipelinesascode.tekton.dev/sha"
+	pacShaUrlAnnotationName  = "pipelinesascode.tekton.dev/sha-url"
+	pacRepoUrlAnnotationName = "pipelinesascode.tekton.dev/repo-url"
+
+	gitCommitShaAnnotationName = "build.appstudio.redhat.com/commit_sha"
+	gitRepoAnnotationName      = "build.appstudio.openshift.io/repo"
```

Comment:
include this annotation also for PaC workflow.

Reply:
ok